packages/webpack-plugin/src/loader.ts

```diff
@@ -28,22 +29,29 @@ export default function StylableWebpackLoader(this: StylableLoaderContext, sourc
         urls,
         unusedImports,
     });
-    /**
-     * NOTICE: order of replacements is coupled with "webpack-entities.ts"
-     * replacement is done from bottom->top.
-     */
-    return `
-${imports.join('\n')}
-export ${varType} namespace = ${getReplacementToken('namespace')};
-export ${varType} classes = ${getReplacementToken('classes')};
-export ${varType} keyframes = ${getReplacementToken('keyframes')}; 
-export ${varType} layers = ${getReplacementToken('layers')};
-export ${varType} stVars = ${getReplacementToken('stVars')}; 
-export ${varType} vars = ${getReplacementToken('vars')}; 
-export ${varType} cssStates = ${getReplacementToken('stc')};
-export ${varType} style = ${getReplacementToken('sts')};
-export ${varType} st = ${getReplacementToken('st')};
-/* JS_INJECT */
-if(import.meta.webpackHot /* HMR */) { import.meta.webpackHot.accept();}
-`;
+
+    return generateStylableJSModuleSource(
```

Comment:
does the order of the generated module still matters? changes to `generateStylableJSModuleSource` template might cause bugs?

Reply:
solved in this PR